### DISCUSSION TITLE
Is there a way to avoid downtime while upgrading "herokuish" package?

### DISCUSSION THREAD

**Discussion by osbre** [dokku/dokku#5209] [Q&A] [answered]
I was upgrading Dokku [using `apt`](https://dokku.com/docs/getting-started/upgrading/#upgrading-using-apt) and noticed ~10 seconds of downtime in my app.

Is there a way to do some sort of "silent" upgrades that don't cause any downtime?

And if not, would that be possible to return a specific HTML page from Nginx to show that "we are doing updates"?

Thank you for your work ❤️

**Comment by josegonzalez** [comment, marked as answer]
There shouldn’t be any downtime during the upgrade process. If you’re seeing downtime, that sounds like a bug that you should report. Please file a ticket for that.
